tests/test_logger.py

```diff
@@ -34,6 +42,24 @@ def test_colorize(self):
             # This is not a tty, so it should not colorize anything
             self.assertEqual(msg, name)
 
+    @patch('sys.stderr.write')
```

Comment:
Python 2 on linux seems to complain about it:
```
ERROR: test_stderr (tests.test_logger.TestLogger)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.13/lib/python2.7/site-packages/mock/mock.py", line 1318, in patched
    patching.__exit__(*exc_info)
  File "/home/travis/virtualenv/python2.7.13/lib/python2.7/site-packages/mock/mock.py", line 1482, in __exit__
    delattr(self.target, self.attribute)
AttributeError: 'file' object attribute 'write' is read-only
```

I think a more appropriate way would be to mock `sys.stderr` object itself with something like StringIO, instead of mocking `write` method specifically. Could you please give it a try?